gmsh_utils/gmsh_IO.py

```diff
@@ -402,6 +406,41 @@ def generate_extract_mesh(self, dims: int, mesh_name: str, mesh_output_dir: str,
 
         gmsh.finalize()
 
+    def set_mesh_size(self, dims: int, mesh_size_list: List[float]) -> None:
```

Comment:
this function sets the a mesh size to all the physical groups, furthermore in this case, the user needs to remember the order of all the physical groups and the length of the mesh_size_list has to be equal to the amount of physical groups, in this case it is easy to make error.

 But what should be possible for the user is to select one physical group (by name) and sets its mesh size.

setting the global mesh size is also important, but this should be unrelated to physical groups.